src/components/Select/components/SelectControl/SelectControl.scss

```diff
@@ -1,93 +1,27 @@
 @use '../../../../../styles/mixins';
 @use '../../../variables';
 
-@mixin button-text-color($color, $hoverColor: $color) {
-    &,
-    &:link,
-    &:visited,
-    &:active,
-    &:focus {
-        color: $color;
-    }
-
-    &:hover {
-        color: $hoverColor;
-    }
-}
-
 $block: '.#{variables.$ns-new}select-control';
+$buttonBlock: '.#{variables.$ns}button';
 
 #{$block} {
-    @include mixins.button-reset();
-    @include button-text-color(var(--yc-color-text-primary));
-    @include mixins.pin($block, $block, var(--yc-select-border-radius));
-
-    box-sizing: border-box;
-    display: inline-flex;
-    align-items: center;
     width: 100%;
     min-width: 34px;
-    border: 1px solid var(--yc-color-line-generic);
```

Comment:
let's add `text-align: right` to place icon correctly.

<img width="123" alt="image" src="https://user-images.githubusercontent.com/10867286/220703767-f8023f65-f3ac-46af-a240-572148b326be.png">